src/credentials/CHIPCert_Internal.h

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <credentials/CHIPCert.h>
+
+namespace chip {
+namespace Credentials {
+
+inline constexpr CharSpan kNetworkIdentityCN           = "*"_span;
+inline constexpr ByteSpan kNetworkIdentitySerialNumber = ByteSpan((uint8_t[1]){ 1 });
```

Comment:
Requiring value `1` prevents ever allowing revocation. We should never force the serial number of the cert to be == 1. The CA should be able to set the serial number as needed.

Reply:
Discussed offline. Added a reference to the "Wi-F Authentication with Per-Device Credentials" spec section to cover the "why" questions.